SurfaceDevCenterManager/Program.cs

```diff
@@ -769,6 +784,52 @@ private static async Task<ErrorCodes> MainAsync(string[] args)
                     }
                 }
             }
+            else if (TranslateOption)
+            {
+                Console.WriteLine("> Translate Option");
+
+                if (PublisherId == null)
+                {
+                    Console.WriteLine("> ERROR: publisherid not specified");
+                    retval = ErrorCodes.TRANSLATE_PUBLISHER_ID_MISSING;
+                }
+
+                if (ProductId == null)
+                {
+                    Console.WriteLine("> ERROR: productid not specified");
+                    retval = ErrorCodes.TRANSLATE_PRODUCT_ID_MISSING;
+                }
+
+                if (SubmissionId == null)
+                {
+                    Console.WriteLine("> ERROR: submissionid not specified");
+                    retval = ErrorCodes.TRANSLATE_SUBMISSION_ID_MISSING;
```

Comment:
Minor: I'm guessing this is intentional, but if all 3 IDs are null, the retval will show error code of the SubmissionID missing.

Reply:
Yes - all three are equally wrong cases to be in